stable/cert-manager-crds/Chart.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v2
+description: A chart that bundles cert-manager CRDs with cert-manager helm chart
+home: https://github.com/mesosphere/kommander
+name: cert-manager-crds
+version: v0.0.1
```

Comment:
I'd think about having the version match the `appVersion` as it will be very clear in `kommander-applications` repo which versions are bundled.

Reply:
@mhrabovcin I like this idea, I'm applying it but also keeping this conversation open till it gets accepted (since we don't follow this pattern for other charts).

Reply:
Given that we now update both properties with CRD i think we should also have one additional make target that will update both, the CRDs and these properties.

Reply:
I see this was already implemented, thanks.